project-base/app/tests/FrontendApiBundle/Functional/Category/ReadyCategorySeoMix/ReadyCategorySeoMixTest.php

```diff
@@ -1,424 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Tests\FrontendApiBundle\Functional\Category\ReadyCategorySeoMix;
-
-use App\DataFixtures\Demo\CategoryDataFixture;
-use App\DataFixtures\Demo\FlagDataFixture;
-use App\DataFixtures\Demo\ParameterDataFixture;
-use App\DataFixtures\Demo\ReadyCategorySeoDataFixture;
-use App\Model\Category\Category;
-use App\Model\Product\Flag\Flag;
-use App\Model\Product\Parameter\Parameter;
-use App\Model\Product\Parameter\ParameterFacade;
-use Shopsys\FrameworkBundle\Component\ArrayUtils\ArraySorter;
-use Shopsys\FrameworkBundle\Component\Translation\Translator;
-use Shopsys\FrameworkBundle\Model\CategorySeo\ReadyCategorySeoMix;
-use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
-use Tests\FrontendApiBundle\Test\GraphQlTestCase;
-
-class ReadyCategorySeoMixTest extends GraphQlTestCase
```

Comment:
maybe we can preserve these tests "just" by replacing the "slug" queries with "categoryByUuidOrUrlSlugQuery"  category queries

Reply:
wow, great idea – tests are back in 415e8637fc